framework/src/partitioner/LibmeshPartitioner.C

```diff
@@ -149,17 +149,18 @@ LibmeshPartitioner::prepareBlocksForSubdomainPartitioner(
   subdomain_partitioner.chunks.clear();
 
   // Insert each chunk
-  for (const auto & group : _subdomain_blocks)
```

Comment:
this should have been fine with the empty default?

Reply:
Review has been addressed